processing/p5.js-web-editor
# Fix/editor is a focus trap

Fixes #3085 

Changes:
Added an Escape key handler to the CodeMirror instance, enabling removal of focus from the editor upon pressing the 'Escape' key.

Steps to check:- 
1) Make focus on editor
2) Press 'Esc' key
3) Press 'Tab' key 
You will see user is able to move to console with tab key

I have verified that this pull request:

* [x] has no test errors (`npm run test`)
* [x] is from a uniquely-named feature branch and is up to date with the  `develop` branch.
* [x] is descriptively named and links to an issue number, i.e. `Fixes #3085`

Demo Video:- 


https://github.com/processing/p5.js-web-editor/assets/100834122/43ab33bc-86d2-4e51-8067-d82bfbff331d
